src/components/ui/UIToolBar/UIToolBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Toolbar, ThemeProvider, createTheme } from '@mui/material';
+import React from 'react';
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#000000',
+    },
+  },
+  components: {
+    MuiToolbar: {
+      styleOverrides: {
+        root: {
+          backgroundColor: 'white',
+          padding: 0,
+          margin: 0,
+          justifyContent: 'space-between',
+          '@media (min-width: 20.25rem)': {
```

Comment:
i think we can use px here, try to research this approach with px and with rem inside media queries